tempest/matrix/C34Matrix.cpp

```diff
@@ -0,0 +1,109 @@
+#include "tempest/matrix/C34Matrix.hpp"
+// Member functions
```

Comment:
I've been forgoing comment headers to separate parts of the class implementations. We can probably drop this comment.